aws/etc/packer/tools/python/stardog/cluster/update_stardog.py

```diff
@@ -0,0 +1,63 @@
+import logging
+import subprocess
+import sys
+
+import stardog.cluster.utils as utils
+
+
+def upload_file(ip, upload_file):
+    scp_opts = "-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
+    cmd = "scp -r %s %s %s:%s" % (scp_opts, upload_file, ip, upload_file)
+    print(cmd)
+    p = subprocess.Popen(cmd, shell=True)
```

Comment:
This next block looks like the run_command in the last script.  If we can't use what is in utils maybe we should move that function there?